src/content/operations/nat-traversal/_index.en.md

```diff
@@ -356,9 +356,29 @@ NAT, or if other applications on the internal network were already using the 450
 ports, the remote ends would not be able to contact gateway C or D over the expected ports.
 {{% /notice %}}
 
-### Double NAT Traversal
+### Double NAT Traversal (scenario 1)
 
-In this case case, A & B cluster gateways have direct reachability over their private
+In this case, clusters C & D are neither reachable on their private IPs (192.168.0.4 and 192.168.0.4) nor
```

Comment:
Shouldn't it be 192.168.128.4 ?

Reply:
Why? The diagram uses 192.168.0.4

Reply:
Private IPs are the node-ips. As per the diagram its 192.168.0.4 (deliberately shown the same IPs as its possible in such deployments) and the private floating-ips are 10.2.0.1 and 10.2.0.2 respectively. The public-ip auto-discovery for such clusters would be the same for both (2.1.1.1).

Reply:
Ooops, I missed the diagram